From 9ec0fb5ec4884d656854f4877840ad6e382e07a3 Mon Sep 17 00:00:00 2001
From: haiqiang yao <haiqiangx.yao@intel.com>
Date: Wed, 9 Sep 2015 17:41:09 +0800
Subject: [PATCH 464/466] ov5693:fix the stretching issue for 192x160 sensor
 setting

Now,the 192x160 sensor setting scales from 2336x1944 to 192x160
and it will cause video preview stretch.

the patch will change the 192x160 sensor setting scaling from
2460x1956 to 192x160.

Change-Id: Ie71535196b95e8033b9e672483180a6991335805
Signed-off-by: haiqiang yao <haiqiangx.yao@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-16828
Reviewed-on: https://icggerrit.ir.intel.com/73307
Reviewed-by: GuanX Wen <guanx.wen@intel.com>
Approver: Jing Tao <jing.tao@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h   | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
index 2cccc53..c2645ec 100755
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
@@ -653,7 +653,7 @@ static struct ov5693_reg const ov5693_368x304[] = {
 
 /*
  * ov5693_192x160 30fps 17ms VBlanking 2lane 10Bit (Scaling)
- DS from 2336x1944
+ DS from 2460x1956
  */
 static struct ov5693_reg const ov5693_192x160[] = {
 	{OV5693_8BIT, 0x3501, 0x7b},
@@ -672,8 +672,8 @@ static struct ov5693_reg const ov5693_192x160[] = {
 	{OV5693_8BIT, 0x380d, 0x80},
 	{OV5693_8BIT, 0x380e, 0x07},
 	{OV5693_8BIT, 0x380f, 0xc0},
-	{OV5693_8BIT, 0x3811, 0x90},
-	{OV5693_8BIT, 0x3813, 0x06},
+	{OV5693_8BIT, 0x3811, 0x40},
+	{OV5693_8BIT, 0x3813, 0x00},
 	{OV5693_8BIT, 0x3814, 0x31},
 	{OV5693_8BIT, 0x3815, 0x31},
 	{OV5693_8BIT, 0x3820, 0x04},
-- 
1.9.1

